pallets/mint_with_fees/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
-name = "pallet-template"
-authors = ["Anonymous"]
-description = "FRAME pallet template for defining custom runtime logic."
+name = "pallet-mint-with-fees"
+authors = ["Erudyx"]
```

Comment:
Who is Erudyx. Put a contact at least, so somebody can message you about this work later on

Reply:
I usually write
`authors     = ["Timothée Delabrouille <https://github.com/tdelabro>"]` You can also give your mail